tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -760,6 +760,9 @@ def test_run_info(self):
             if k in ["source_name", "source_type", "source_version", "name", "entry_point_name"]:
                 continue
 
+            if k == "run_uuid":
+                k = "run_id"
```

Comment:
Are we going to rename the `run_uuid` column to `run_id`?

Reply:
I wasn't planning on it, since there's really no need

Reply:
Got it :)